### DISCUSSION TITLE
Serialize data and send as a parameter to the next page

### DISCUSSION THREAD

**Discussion by gitmatsika** [flet-dev/flet#2761] [Q&A]
Environment : flet 0.20

Situation
data = {"admin" : 1, "auth" : 1}
self.page.go(f"/login/{data}")

I have a dictionary that i want to pass as a parameter to the called page

When my new page open i am getting the passed parameter as a string 
How do i serialize

**Comment by 1mrnewton** [comment]
`page.go` method accepts kwargs, so instead of the dict, you can pass as kwargs. 
Example:
```python
page.go("/login", admin=1, auth=1)
```

You can then use `urlib.parse` to extract as dict inside your `on_route_change` handler.

**Comment by gitmatsika** [comment]
Thanks, I'll check it out.
